FAIL: balance_test.go:719: testReplicaCheckerSuite.TestStorageThreshold

This test failed three times when releasing 2.1.0-beta.

```
go test github.com/pingcap/pd/table github.com/pingcap/pd/pkg/typeutil github.com/pingcap/pd/pkg/etcdutil github.com/pingcap/pd/pkg/logutil github.com/pingcap/pd/pkg/metricutil github.com/pingcap/pd/pd-client github.com/pingcap/pd/server github.com/pingcap/pd/server/cache github.com/pingcap/pd/server github.com/pingcap/pd/server/schedule github.com/pingcap/pd/server github.com/pingcap/pd/server/schedulers github.com/pingcap/pd/server/api github.com/pingcap/pd/server github.com/pingcap/pd/server/core github.com/pingcap/pd/server
ok  	github.com/pingcap/pd/table	0.008s
ok  	github.com/pingcap/pd/pkg/typeutil	0.005s
ok  	github.com/pingcap/pd/pkg/etcdutil	2.303s
ok  	github.com/pingcap/pd/pkg/logutil	0.005s
ok  	github.com/pingcap/pd/pkg/metricutil	0.008s
ok  	github.com/pingcap/pd/pd-client	93.461s
ok  	github.com/pingcap/pd/server	22.661s
ok  	github.com/pingcap/pd/server/cache	2.004s
ok  	github.com/pingcap/pd/server/schedule	0.010s
time="2018-06-29T15:45:57Z" level=info msg="lost the store 3, maybe you are recovering the PD cluster."

----------------------------------------------------------------------
FAIL: balance_test.go:719: testReplicaCheckerSuite.TestStorageThreshold

balance_test.go:737:
    testutil.CheckTransferPeer(c, rc.Check(region), schedule.OpReplica, 1, 4)
/go/src/github.com/pingcap/pd/pkg/testutil/operator_check.go:66:
    c.Assert(op.Step(2).(schedule.RemovePeer).FromStore, check.Equals, sourceID)
... obtained uint64 = 0x2
... expected uint64 = 0x1

OOPS: 28 passed, 1 FAILED
--- FAIL: TestSchedulers (0.21s)
FAIL
FAIL	github.com/pingcap/pd/server/schedulers	0.216s
ok  	github.com/pingcap/pd/server/api	109.497s
ok  	github.com/pingcap/pd/server/core	0.032s
Makefile:43: recipe for target 'basic_test' failed
make: *** [basic_test] Error 1
Exited with code 2
```

--- a/server/schedulers/balance_test.go
+++ b/server/schedulers/balance_test.go
@@ -724,8 +724,10 @@ func (s *testReplicaCheckerSuite) TestStorageThreshold(c *C) {
 
 	tc.AddLabelsStore(1, 1, map[string]string{"zone": "z1"})
 	tc.UpdateStorageRatio(1, 0.5, 0.5)
+	tc.UpdateStoreRegionSize(1, 500*1024*1024)
 	tc.AddLabelsStore(2, 1, map[string]string{"zone": "z1"})
-	tc.UpdateStorageRatio(1, 0.1, 0.9)
+	tc.UpdateStorageRatio(2, 0.1, 0.9)
+	tc.UpdateStoreRegionSize(2, 100*1024*1024)
 	tc.AddLabelsStore(3, 1, map[string]string{"zone": "z2"})
 	tc.AddLabelsStore(4, 0, map[string]string{"zone": "z3"})
 

